keys/generate-key-pair.sh

```diff
@@ -2,89 +2,105 @@
 # This script will generate a Key-Pair for Owner Attestation.
 
 if [[ "$1" == "-h" || "$1" == "--help" ]]; then
-    cat << EndOfMessage
+  cat << EndOfMessage
 Usage: ${0##*/} [<encryption-keyType>]
 
 Arguments:
   <encryption-keyType>  The type of encryption to use when generating owner key pair (ecdsa256, ecdsa384, rsa, or all). Will default to all.
+
+Option Environment Variables:
+  countryName - The country the user resides in. Necessary information for keyCertificate generation.
```

Comment:
Remember, these env vars should be all caps with underscores separating the words in the names.